libs/aws/langchain_aws/chat_models/bedrock_converse.py

```diff
@@ -314,6 +314,24 @@ class Joke(BaseModel):
     config: Any = None
     """An optional botocore.config.Config instance to pass to the client."""
 
+    guardrail_config: Optional[Dict[str, Any]] = Field(None, alias="guardrails")
+    """Configuration information for a guardrail that you want to use in the request."""
+
+    additional_model_request_fields: Optional[Dict[str, Any]] = None
```

Comment:
Is there a reason why `max_tokens`, `stop_sequences` etc. being passed as class variables and not a part of the `inference_config` dict? If its an `inference_config` dict, you can limit them to these specified values (`max_tokens`, `stop_sequences` etc.).

We have `additional_model_request_fields` and `additional_model_response_field_paths` here which are the inputs to the converse API call as well. So having `inference_config` would be in line with the converse API call and these other fields.

Reply:
Also, if its `inference_config` instead of the individual inference parameters, then any changes to add or remove these parameters from `inferenceConfig` in the converse API call wouldn't have to be handled by us.

Reply:
this is done for a nicer devx and for interoperatbility. max_tokens, stop_sequences, [few other params](https://github.com/langchain-ai/langchain/issues/20085) are standardized across all the main models, making it extra easy to swam models in your code

Reply:
These properties are pretty common in all providers, it's ok to have these at the root. Alternately, `inferenceConfig` is also supported if users want to pass them in that fashion.